document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/service/ApiClientService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.document.generator.prosecution.service;
+
+import java.util.UUID;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.http.ApiKeyHttpClient;
+import uk.gov.companieshouse.api.http.HttpClient;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+@Component
+public class ApiClientService {
```

Comment:
Are there unit tests for this service?

Reply:
We have agreed in person that unit tests are not necessary for this class as I have brought in the api-sdk-manager-java library into the service to handle calls to the SDK